content/versions-comparison/4.0-news.json

```diff
@@ -1,55 +1,48 @@
 [
   {
-    "title":"Assets structuration with families and attributes 🖼️",
+    "title":"Asset Manager for your digital assets 🖼️",
     "domains": ["Asset","Governance"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#more-flexibility-to-manage-your-assets"
   },
   {
-    "title":"Asset data enrichment 🖼️",
-    "domains": ["Asset","Quality"],
-    "editions": ["EE"],
-    "since-version": "4.0",
-    "link": "/pim/v4/updates/asset-manager.html#more-flexibility-to-manage-your-assets"
-  },
-  {
-    "title":"Asset data contextualization by locale and channel",
+    "title":"Contextualize asset content by locale and channel",
     "domains": ["Asset","Quality"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#contextualize-your-asset-information"
   },
   {
-    "title":"External media used as product asset 🖼️",
+    "title":"Easily retrieve 3rd-party assets 🖼️",
     "domains": ["Asset","Governance"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#more-flexibility-to-manage-your-assets"
   },
   {
-    "title":"Asset completeness 🖼️",
+    "title":"Completeness on assets 🖼️",
     "domains": ["Asset","Quality"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#permissions-completeness"
   },
   {
-    "title":"Flexible and automatized asset transformations 🤖",
+    "title":"New asset transformations 🤖",
```

Comment:
What is really new here compared to the PAM, it's the fact that it is flexible and automatized (it was poorly automatized before).